<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>import_audio_file</class>
 <widget class="QMainWindow" name="import_audio_file">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Import Audio</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>230</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Select File:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="import_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>230</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Import</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>230</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>230</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Audio Name:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="select_file_button">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>29</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="audio_name_box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>35</number>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="file_path_display">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>205</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <action name="action_configure_audio_devices">
   <property name="text">
    <string>Configure Audio Devices</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
